src/main/java/utils/helpers/HelperFunctions.java

```diff
@@ -0,0 +1,34 @@
+package utils.helpers;
+
+import utils.exceptions.IllegalValueException;
+
+import java.time.LocalDateTime;
+
+public class HelperFunctions {
```

Comment:
I like how you have a file to store helper functions. Are they the same as components?